app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/data/GetMemberActionsUseCase.kt

```diff
@@ -0,0 +1,106 @@
+package com.hedvig.android.feature.help.center.data
+
+import arrow.core.Either
+import arrow.core.raise.either
+import com.apollographql.apollo3.ApolloClient
+import com.hedvig.android.apollo.safeExecute
+import com.hedvig.android.apollo.toEither
+import com.hedvig.android.core.common.ErrorMessage
+import com.hedvig.android.featureflags.FeatureManager
+import com.hedvig.android.featureflags.flags.Feature
+import com.hedvig.android.logger.logcat
+import kotlinx.coroutines.flow.first
+import kotlinx.serialization.Serializable
+import octopus.MemberActionsQuery
+
+internal interface GetMemberActionsUseCase {
+  suspend fun invoke(): Either<ErrorMessage, MemberAction>
+}
+
+internal class GetMemberActionsUseCaseImpl(
+  private val apolloClient: ApolloClient,
+  private val featureManager: FeatureManager,
+) : GetMemberActionsUseCase {
+  override suspend fun invoke(): Either<ErrorMessage, MemberAction> {
+    return either {
+      val result = apolloClient.query(MemberActionsQuery())
+        .safeExecute().toEither(::ErrorMessage).onLeft {
+          logcat { "Cannot load memberActions" }
+        }.bind()
+      val isCoInsuredFeatureOn = featureManager.isFeatureEnabled(Feature.EDIT_COINSURED).first()
+      val isMovingFeatureOn = featureManager.isFeatureEnabled(Feature.MOVING_FLOW).first()
+      val isConnectPaymentFeatureOn = featureManager.isFeatureEnabled(Feature.PAYMENT_SCREEN).first()
+      MemberAction(
+        isCancelInsuranceEnabled = result.currentMember.memberActions?.isCancelInsuranceEnabled ?: false,
+        isConnectPaymentEnabled = isConnectPaymentFeatureOn && result.currentMember.memberActions?.isConnectPaymentEnabled ?: false,
+        isEditCoInsuredEnabled = isCoInsuredFeatureOn && result.currentMember.memberActions?.isEditCoInsuredEnabled ?: false,
+        isMovingEnabled = isMovingFeatureOn && result.currentMember.memberActions?.isMovingEnabled ?: false,
+        isTravelCertificateEnabled = result.currentMember.memberActions?.isTravelCertificateEnabled ?: false,
+        sickAbroadAction = result.currentMember.memberActions?.sickAbroadAction.toSickAbroadAction(),
+        firstVetAction = result.currentMember.memberActions?.firstVetAction?.toVetAction(),
+      )
+    }
+  }
+}
+
+internal data class MemberAction(
+  val isCancelInsuranceEnabled: Boolean?,
+  val isConnectPaymentEnabled: Boolean?,
+  val isEditCoInsuredEnabled: Boolean?,
+  val isMovingEnabled: Boolean?,
+  val isTravelCertificateEnabled: Boolean?,
+  val sickAbroadAction: MemberActionWithDetails.SickAbroadAction?,
+  val firstVetAction: MemberActionWithDetails.FirstVetAction?,
+)
+
+internal sealed interface MemberActionWithDetails {
+  data class SickAbroadAction(
+    val partners: List<DeflectPartner>?,
+  ) : MemberActionWithDetails
+
+  data class FirstVetAction(
+    val sections: List<FirstVetSection>,
+  ) : MemberActionWithDetails
+}
```

Comment:
What does it mean for all these to be nullable here?
Is there something stopping us from doing this instead

```suggestion
internal data class MemberAction(
  val isCancelInsuranceEnabled: Boolean,
  val isConnectPaymentEnabled: Boolean,
  val isEditCoInsuredEnabled: Boolean,
  val isMovingEnabled: Boolean,
  val isTravelCertificateEnabled: Boolean,
  val sickAbroadAction: MemberActionWithDetails.SickAbroadAction?,
  val firstVetAction: MemberActionWithDetails.FirstVetAction?,
)

internal sealed interface MemberActionWithDetails {
  data class SickAbroadAction(
    val partners: List<DeflectPartner>,
  ) : MemberActionWithDetails

  data class FirstVetAction(
    val sections: List<FirstVetSection>,
  ) : MemberActionWithDetails
}
```
So that the booleans are either true or false.
And the other two are either null, which should show nothing, or be non-null with content. We might even want to use NonEmptyList in there instead of List? Perhaps?